CentralHub.Api/Controllers/MeasurementController.cs

```diff
@@ -15,21 +15,38 @@ public sealed class MeasurementController : ControllerBase
 
     private readonly IMeasurementRepository _aggregatorRepository;
 
+    private readonly ITrackerRepository _trackerRepository;
+
     public MeasurementController(
         ILogger<MeasurementController> logger,
-        IMeasurementRepository aggregatorRepository)
+        IMeasurementRepository aggregatorRepository,
+        ITrackerRepository trackerRepository)
     {
         _logger = logger;
         _aggregatorRepository = aggregatorRepository;
+        _trackerRepository = trackerRepository;
     }
 
     [HttpPost("add")]
-    public async Task AddMeasurements(AddMeasurementsRequest addMeasurementsRequest, CancellationToken token)
+    public async Task<AddMeasurementResponse> AddMeasurements(AddMeasurementsRequest addMeasurementsRequest, CancellationToken token)
```

Comment:
Async task without async in the name, me not understand why

Reply:
idk, its like the others...